OvieMudi/Banka
# #165662101 consume the sign-in endpoint

#### What does this PR do?
 1. Allows user sign in via the auth endpoint
 2. Adds script and HTML for user signout

#### Description of Task to be completed?
Ensure the signup form at the URL below can sign a user when supplied with valid credentials

https://oviemudi.github.io/Banka/UI/signup

#### How should this be manually tested?
- clone the repo ` git clone git@github.com:OvieMudi/Banka.git `

- on your terminal run ` cd Banka `

- run ` git ft-165662101-consume-signin-api  `

- run ` npm install `

- run ` npm run dev:start `

- make a GET request to  ` localhost:3000/signup.html`

- supply valid credentials as instructed by the form validation

- submit form

NB see screenshot below on the message the form displays on invalid entry
#### Any background context you want to provide?
This feature will allow 
users to easily log out from the Banka service
#### What are the relevant pivotal tracker stories?
[#165662101 ](https://www.pivotaltracker.com/story/show/165662101 )
#### Screenshots (if appropriate)
![signin error](https://user-images.githubusercontent.com/42499892/56869918-5844ed80-69ff-11e9-9445-e435b56c9bc9.PNG)
